images/metering-ansible-operator/roles/meteringconfig/tasks/configure_metastore.yml

```diff
@@ -0,0 +1,9 @@
+- name: Get metastore secret
+  block:
+    - name: Get Metastore storage credentials secret
+      k8s_info:
+        api_version: v1
+        kind: Secret
+        name: "{{ meteringconfig_spec.hive.spec.config.db.secretName | b64decode }}"
```

Comment:
```suggestion
        name: "{{ meteringconfig_spec.hive.spec.config.db.secretName }}"
```

Reply:
We should reference the `meteringconfig_spec_overrides` dictionary instead of the `meteringconfig_spec`.